model-optimizer/extensions/front/tf/pad_tf_to_pad.py

```diff
@@ -34,6 +35,9 @@ def find_and_replace_pattern(self, graph: Graph):
                 # the input with fill value is an optional third input in TF
                 if not tfpad.in_port(2).disconnected():
                     tfpad.in_port(2).get_connection().set_destination(new_pad.in_port(3))
+                else:
```

Comment:
BTW, why the check here is `else:`? May be it should be: `if not tfpad.in_port(3).disconnected():`

Reply:
Thanks. But no, it should not. It will cause errors like `Input port with index 3 is not in node`

Reply:
Ok, I see. But does the model provided in the 41622 works correctly after your change?

Reply:
I will check the model

Reply:
The model does not work with my changes. 
Thanks for review. Will work on it a bit more.